language/tools/test_generation/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "test_generation"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+rand = "0.6.5"
+log = "*"
```

Comment:
This is inconsistent with the previous line. You may as well specify a lower bound here.